components/text-area/src/index.js

```diff
@@ -43,31 +43,25 @@ const TextAreaField = styled('textarea')(
   }),
 );
 
-const TextArea = ({ className, ...props }) => (
-  <Label className={className} error={props.meta.touched && props.meta.error}>
-    <LabelText>{props.children}</LabelText>
-    {props.hint && <HintText>{props.hint}</HintText>}
-    {props.meta.touched &&
-      props.meta.error && <ErrorText>{props.meta.error}</ErrorText>}
-    <TextAreaField
-      type="text"
-      rows="5"
-      error={props.meta.touched && props.meta.error}
-      {...props.input}
-    />
+const TextArea = ({
+  children, hint, meta, ...input
```

Comment:
you need to add in `input` here as an explicit prop

as you can see above in the previous code, `TextAreaField` was getting `{...prop.input}`

so this should be `children, hint, meta, input, ...props`

Reply:
Oops, good spot! Completely missed the .input at the end of ...props when making this change